installer/dockerfiles/alpine-main/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.8.13-alpine3.15
+FROM python:3.8.16-alpine3.17
```

```suggestion
FROM public.ecr.aws/docker/library/python:3.8.16-alpine3.17
```

We should use the equivalent image in the ECR Public Gallery